ethernaut/vault/contracts/Vault.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract Vault {
+    bool public locked;
+    bytes32 private password;
```

Comment:
for an additional challenge, consider storing private data within a dynamically sized array or a mapping! you'll then need to use abi.encodePacked in additional to getStorageAt